recipe/build.sh

```diff
@@ -12,6 +12,17 @@ rm -rf pyproject.toml
 # uncomment to debug cmake build
 # export CMAKE_VERBOSE_MAKEFILE=1
 
+export CMAKE_FIND_DEBUG_MODE=1
+export USE_NUMPY=1
+
+# # Force failure if numpy can't be found
+# sed -i.bak 's/Python COMPONENTS Interpreter OPTIONAL_COMPONENTS/Python COMPONENTS Interpreter/g' cmake/Dependencies.cmake
```

Comment:
You should leave this one. It helps fail eart

Reply:
Sorry about that; I wasn't planning on merging the debug commits here anyway; I just wanted to ensure we don't need to hack CMake itself. I agree that we should keep the failsafe. Despite removing that now, I checked the logs an it does show
```
  --   USE_NUMPY             : ON
```

Reply:
Yeah. I could use your help in understanding why aarch64 isn’t detecting things…..

Reply:
I removed that line from main since I felt it was too fragile. 

But I think it is useful for debugging. 

Aarch64 failures really puzzle me…..